### PR TITLE
Remove duplicated constraint values from Scenario model and fix tests

### PR DESCRIPTION
**Pull request by elsiel23**

### CODE REVIEW

File: src/planscape/plan/views.py
Comment (strangeties) [inline src/planscape/plan/views.py:286]: nit: I'd prefer seeing 'max_budget', 'max_treatment_area_ratio', etc. as constants defined in a single place

File: src/planscape/plan/views.py
Comment (strangeties) [inline src/planscape/plan/views.py:208]: Are any tests for create_project and update_project affected?

Comment (elsiel23) [reply src/planscape/plan/views.py:208]: No changes in test behavior were intended.

File: src/planscape/plan/views.py
Comment (elsiel23) [reply src/planscape/plan/views.py:286]: I'm not sure what you mean since this is a HTTP method and not a class. Can you clarify?

Comment (strangeties) [reply src/planscape/plan/views.py:286]: yea:
I mean 'max_budget' is defined in-line in two places, which increases the risk of typos, and if the text is changed, it needs to be changed in two locations.

I'd prefer to say
_MAX_BUDGET_URL_PARAM = 'max_budget' # near top of file
body.get(_MAX_BUDGET_URL_PARAM, None) # in the function

wdyt? I'll approve since this suggestion shouldn't block the pr.

**Review by strangeties — APPROVED** [review]

File: src/planscape/plan/views.py
Comment (elsiel23) [reply src/planscape/plan/views.py:286]: oh yes! i see now. i can make that change before merging. thanks for the explanation!

**Pull request merged**
